sw/host/provisioning/orchestrator/configs/skus/sival.hjson

```diff
@@ -10,6 +10,7 @@
   target_lc_state: "prod",
   otp: "sival",
   perso_bin: "sw/device/silicon_creator/manuf/base/binaries/ft_personalize_{sku}_{target}.signed.bin",
+  # owner_fw_boot_str: "Bare metal PASS!"
```

Comment:
Is this meant to be commented out?

Reply:
Yes, currently the bare metal image for the sival SKU is not signed properly so it boot faults. This should be cleaned up in a follow up.